js/spreadsheet.js

```diff
@@ -351,14 +351,101 @@ var spreadsheet = function(options) {
 };
 
 // Search
-var searchFiled = document.getElementById('search-field');
-Handsontable.dom.addEvent(searchFiled, 'keyup', function (event) {
-  var queryResult = hot.search.query(this.value);
-  var resultsCount = queryResult.length;
-  $('.find-results').html(resultsCount + ' found');
-  hot.render();
+var searchField = document.getElementById('search-field');
+
+// Prevent cmd+f default behaviour and use our find
+window.addEventListener("keydown", function (event) {
+  var ctrlDown = (event.ctrlKey || event.metaKey) && !event.altKey;
+
+  if (!ctrlDown) {
+    return;
+  }
+
+  if (event.keyCode === 114 || (ctrlDown && event.keyCode === 70)) { 
+    event.preventDefault();
+    hot.deselectCell();
```

Comment:
I think this'll trigger an error if the user is looking at the data source list, because HOT isn't initialised

Reply:
`TypeError: Cannot read property 'deselectCell' of undefined`